### ISSUE TITLE
Intermediate Timer Events Import

### ISSUE THREAD

**Issue by velkymx** [ProcessMaker/modeler#448]
Imported one of my processes and it did this. There should be an intermediate timer event where those two flow arrows meet. 

![image](https://user-images.githubusercontent.com/535300/60202150-d4d92a00-97fe-11e9-8ffb-80f5d1e81c13.png)

**Comment by PMQABen** [comment]
[q_a_test-_start_event.json.txt](https://github.com/ProcessMaker/modeler/files/3331305/q_a_test-_start_event.json.txt)
Process attached (remove the .txt)

**Comment by chipit24** [comment]
@PMQABen I downloaded that process (had to rename `.json` to `.spark` for processmaker to accept the file) and didn't have any issues with importing or rendering:

<img width="521" alt="Screen Shot 2019-06-26 at 3 29 26 PM" src="https://user-images.githubusercontent.com/7561061/60208920-7d02e900-9827-11e9-8ddf-46158a987a13.png">

I tested with the latest `develop` branch of both `modeler` and `processmaker` (and also just latest `processmaker` only without npm linking modeler). I also tried a few simpler processes using pools, lanes, and the intermediate timer event, and had no issues. I also checkout out the `beta6` commit and couldn't reproduce it either. @PMQABen @velkymx What version of processmaker was this bug discovered on? 

I see from your error that it says `<modeler-intermediate-catch-timer-event>`, but the modeler is rendering the node as `<processmaker-modeler-intermediate-catch-timer-event>`. I didn't find any references to just `modeler-intermediate-catch-timer-event`; so it seems whatever was causing the bug is no longer present with the latest versions.
